pushmsixscript/src/pushmsixscript/manifest.py

```diff
@@ -43,9 +43,9 @@ def _readline_generator(fp):
         with zf.open(app_manifest) as f:
             config = configparser.ConfigParser()
             config.read_file(_readline_generator(f))
-            version = config.get("App", "Version")
-            build_id = config.get("App", "BuildId")
-            code_name = config.get("App", "CodeName")
+            version = config.get("App", "Version", fallback="unknown")
+            build_id = config.get("App", "BuildId", fallback="unknown")
+            code_name = config.get("App", "CodeName", fallback="unknown")
```

Comment:
Should we fallback to `config.get("App", "Name")` ?

Reply:
Good idea. Thanks!